### PR TITLE
Implement PointCoord and PointSize shader attributes

### PR DESCRIPTION
**Pull request by ReinUsesLisp**

### CODE REVIEW

File: Ryujinx.Graphics/Gal/Shader/GlslDecl.cs [hunk 1]
```diff
@@ -249,11 +252,15 @@ private void Traverse(ShaderIrNode[] Nodes, ShaderIrNode Parent, ShaderIrNode No
...
                 case ShaderIrOperAbuf Abuf:
                 {
-                    //This is a built-in input variable.
-                    if (Abuf.Offs == VertexIdAttr ||
-                        Abuf.Offs == InstanceIdAttr ||
-                        Abuf.Offs == FaceAttr ||
-                        Abuf.Offs == LayerAttr)
+                    //This is a built-in variable.
+                    if (Abuf.Offs == LayerAttr       ||
+                        Abuf.Offs == PointSizeAttr   ||
```
Comment (ReinUsesLisp) [inline Ryujinx.Graphics/Gal/Shader/GlslDecl.cs]: I'm not sure if these attributes belong here

File: Ryujinx.Graphics/Gal/Shader/GlslDecl.cs [hunk 2]
```diff
@@ -249,11 +252,15 @@ private void Traverse(ShaderIrNode[] Nodes, ShaderIrNode Parent, ShaderIrNode No
...
+                        Abuf.Offs == PointCoordAttrX ||
+                        Abuf.Offs == PointCoordAttrY ||
+                        Abuf.Offs == VertexIdAttr    ||
+                        Abuf.Offs == InstanceIdAttr  ||
+                        Abuf.Offs == FaceAttr
+                        )
```
Comment (gdkchan) [inline Ryujinx.Graphics/Gal/Shader/GlslDecl.cs]: nit: Imo `)` should be after `FaceAttr`. Other than that lgtm.

Comment (ReinUsesLisp) [reply Ryujinx.Graphics/Gal/Shader/GlslDecl.cs]: It was not intended

**Comment by gdkchan** [comment]
thanks 👍

**Pull request merged**